client/layout/guided-tours/tours/editor-insert-menu-tour.js

```diff
@@ -36,8 +38,19 @@ class RepositioningStep extends Step {
 		clearInterval( this.interval );
 	}
 
+	render() {
+		if ( this.props.isEditorModeVisual ) {
+			return super.render();
+		}
+		return null;
+	}
+
 }
 
+const ConnectedStep = connect( state => ( {
+	isEditorModeVisual: 'html' !== getPreference( state, 'editor-mode' ),
```

Comment:
I don't feel overly strong about it, but given how we'll collaborate with the tinymce team on the new WordPress Core editor I'd be surprised if that happened reasonably soon